runtime/compiler/ilgen/J9ByteCodeIlGenerator.hpp

```diff
@@ -205,6 +205,7 @@ class TR_J9ByteCodeIlGenerator : public TR_IlGenerator, public TR_J9ByteCodeIter
    void         storeFlattenableInstance(int32_t);
    void         storeFlattenableInstanceWithHelper(int32_t);
    void         storeStatic(int32_t);
+   TR::Node*    narrowIntStore(TR::Node *value, TR::SymbolReference *symRef);
```

Comment:
Just a very minor comment: `narrowIntStore` can be invoked if the type is `TR::Int32` and no action will be taken. I wonder if `narrowIntStore` should be updated to something like `narrowIntStoreIfRequired` to reflect that not all integral type are going through narrowing conversion.